cmd/hello/main.go

```diff
@@ -11,6 +11,10 @@ func main() {
 	a := app.New()
 	w := a.NewWindow("Hello")
 
+	w2 := a.NewWindow("Second")
```

Comment:
Not sure when this slipped in?

Reply:
Me neither. Strange. Will get that removed.